tests/asserts/final/publication/metadata.py

```diff
@@ -0,0 +1,51 @@
+from layman import app
+from layman.layer.micka import csw as csw_util
+from test_tools import process_client, assert_util
+
+METADATA_PROPERTIES = {
+    'abstract',
+    'extent',
+    'graphic_url',
+    'identifier',
+    'layer_endpoint',
+    'language',
+    'organisation_name',
+    'publication_date',
+    'reference_system',
+    'revision_date',
+    # 'spatial_resolution',  # It is not updated for vector layer updated as raster one (https://trello.com/c/hRkFNuXP)
```

Comment:
Link to private source seems not suitable. As well as link to source that will probably unavailable in a month or two.
Instead of link, I would add something like `probably because of bug on Micka`